src/averageAges.js

```diff
@@ -15,11 +15,13 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const mens = whatTheSex(people, 'm');
+
+  const filteredMens = mens.filter(men => (
+    (!century || Math.ceil(men.died / 100) === century)
+  ));
```

Comment:
You filtered this array twice, but we could do it only once. Let's move check for `century` inside `filter` method